WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -147,26 +94,106 @@ struct ProductCreationAIStartingInfoView: View {
 }
 
 private extension ProductCreationAIStartingInfoView {
+    func textField(with proxy: ScrollViewProxy) -> some View {
+        TextField(Localization.placeholder, text: $viewModel.features, axis: .vertical)
+            .id(Constant.textFieldID)
+            .bodyStyle()
+            .foregroundStyle(.secondary)
+            .lineLimit(Constant.textFieldMinLineLength...)
+            .padding(insets: Layout.messageContentInsets)
+            .focused($editorIsFocused)
+            // Scrolls to the "TextField" view with a smooth animation while typing.
+            .onChange(of: viewModel.features) { _ in
+                scrollToTextField(using: proxy)
+            }
+            // Scrolls to the "TextField" view with a smooth animation when the editor is focused in a small screen.
+            .onChange(of: editorIsFocused) { isFocused in
+                if isFocused {
+                    scrollToTextField(using: proxy)
+                }
+            }
+            .overlay(
+                RoundedRectangle(cornerRadius: Layout.cornerRadius)
+                    .stroke(editorIsFocused ? Color(.brand) : Color(.separator))
+            )
+    }
+
+    @ViewBuilder
+    var photoSection: some View {
+        switch viewModel.imageState {
+        case .empty:
+            readTextFromPhotoButton
+                .mediaSourceActionSheet(showsActionSheet: $viewModel.isShowingMediaPickerSourceSheet, selectMedia: { source in
+                    Task { @MainActor in
+                        await viewModel.selectImage(from: source)
+                    }
+                })
+        case .loading, .success:
+            PackagePhotoView(title: Localization.photoSelected,
+                             imageState: viewModel.imageState,
+                             onTapViewPhoto: {
+                viewModel.didTapViewPhoto()
+            },
+                             onTapReplacePhoto: {
+                viewModel.didTapReplacePhoto()
+            },
+                             onTapRemovePhoto: {
+                viewModel.didTapRemovePhoto()
+            })
+            .clipShape(
+                .rect(
+                    bottomLeadingRadius: Layout.cornerRadius,
+                    bottomTrailingRadius: Layout.cornerRadius
+                )
+            )
```

Comment:
I think we should update this based on the new design. We no longer need to clip only the bottom corners.

| Design | Our code |
|--------|--------|
| <img width="397" alt="Screenshot 2024-07-24 at 10 05 46 AM" src="https://github.com/user-attachments/assets/af54c08a-45e7-493d-b8e0-8cb0c3bca552"> | <img width="397" alt="Screenshot 2024-07-24 at 10 05 46 AM" src="https://github.com/user-attachments/assets/64cb3cab-6810-43f6-93f1-f3d4b8c96048"> |

Reply:
Good catch! I updated the corners in c8b9e82165c09750a04d1dfefda23d89a85faeb4.